web-test-runner.config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {playwrightLauncher} from '@web/test-runner-playwright';
+import {jasmineTestRunnerConfig} from 'web-test-runner-jasmine';
+
+export default {
+  ...jasmineTestRunnerConfig(),
+  nodeResolve: true,
+  files: ['**/*_test.js', '!node_modules/', '!.wireit/'],
```

Comment:
Optional: thoughts on matching `*test.js` instead? 

We have some mixed `file_test.ts` and `file.test.ts` naming conventions that need to be resolved (although the `*.test.ts` ones aren't exported, so this will be fine for now)

Reply:
Done